### PR TITLE
add server URL to the QR code

### PR DESCRIPTION
**Pull request by jiabaow**

### CODE REVIEW

File: fe1-web/components/eventList/LaoProperties.tsx [hunk 1]
```diff
@@ -10,21 +10,23 @@ import { useSelector } from 'react-redux';
 import ParagraphBlock from 'components/ParagraphBlock';
 import { Lao } from 'model/objects';
 import { makeCurrentLao } from 'store/reducers';
+import PropTypes from 'prop-types';
 import QRCode from '../QRCode';
 
-function renderProperties(lao: Lao) {
+function renderProperties(lao: Lao, url: string) {
   const creationDateString = lao.creation.timestampToString();
+  const value = lao.id.toString().concat(' ').concat(url);
```
Comment (pierluca) [inline fe1-web/components/eventList/LaoProperties.tsx:18]: Reminder: there was a discussion this semester to put this in a JSON object and have it documented in JSON-Schema.
I don't remember where or when it happened, but I think the conclusion (robustness-wise) was a good one. 😉

Comment (jiabaow) [reply fe1-web/components/eventList/LaoProperties.tsx:18]: 👌🏻I will change this.

**Review by GabrielFleischer — APPROVED** [review]
I am not proficient with node, but to me, this look great !

We still need to do a cross-system test before merging the feature on main.

**Review by Xelowak — APPROVED** [review]
Nice job, thanks a lot ! I just made a suggestion for the documentation.

File: fe1-web/components/eventList/LaoProperties.tsx [hunk 2]
```diff
@@ -10,21 +10,28 @@ import { useSelector } from 'react-redux';
...
 import QRCode from '../QRCode';
 
-function renderProperties(lao: Lao) {
+function renderProperties(lao: Lao, url: string) {
   const creationDateString = lao.creation.timestampToString();
+  const obj = new ConnectToLao({
+    server: url,
+    lao: lao.id.toString(),
+  });
+  const connectToLao = ConnectToLao.fromJson(obj);
```
Comment (pierluca) [inline fe1-web/components/eventList/LaoProperties.tsx:23]: why would we do a `ConnectToLao.fromJson` here if we have already created the `ConnectToLao` object?

File: fe1-web/parts/connect/ConnectOpenScan.tsx [hunk 3]
```diff
@@ -28,7 +28,9 @@ const ConnectOpenScan = ({ navigation }: IPropTypes) => {
     console.log(data);
     if (data) {
       setQrWasScanned(true);
-      navigation.navigate(STRINGS.connect_confirm_title, { laoIdIn: data });
+      const obj = JSON.parse(data);
```
Comment (pierluca) [inline fe1-web/parts/connect/ConnectOpenScan.tsx:31]: why not validate the object with `ConnectToLao.fromJson` ? 
It seems that we're actually never using the JSON Schema to validate what we scan.
And if the validation fails, a toast might be helpful :)

**Review by pierluca — CHANGES_REQUESTED** [review]
Let me know if you have time to do the changes, otherwise I can approve "blindly"

**Review by pierluca — APPROVED** [review]
👍

**Pull request merged**